tests/MongoDB.Driver.Core.Tests/Core/Operations/CreateIndexesOperationTests.cs

```diff
@@ -67,44 +67,6 @@ public void constructor_should_throw_when_messageEncoderSettings_is_null()
             argumentNullException.ParamName.Should().Be("messageEncoderSettings");
         }
 
-        [Theory]
-        [ParameterAttributeData]
-        public void CreateOperation_should_return_expected_result(
```

Comment:
Another reason to keep the CreateOperation helper method in CreateIndexesOperation, so that we can test operation creation.

I would reinstate this test, but removing the `isCommandSupported` test method parameter.

Reply:
Done.